pkg/reader/ccip.go

```diff
@@ -775,6 +775,81 @@ func (r *ccipChainReader) GetContractAddress(contractName string, chain cciptype
 	return addressBytes, nil
 }
 
+// GetLinkPriceUSD gets the LINK price in USD from the FeeQuoter contract on the destination chain.
+func (r *ccipChainReader) GetLinkPriceUSD(ctx context.Context) (cciptypes.BigInt, error) {
+	// Exit without an error if we cannot read the destination.
+	if err := validateExtendedReaderExistence(r.contractReaders, r.destChain); err != nil {
+		return cciptypes.BigInt{}, fmt.Errorf("validate extended reader existence: %w", err)
+	}
+
+	feeQuoterCfg, err := r.getDestFeeQuoterStaticConfig(ctx)
```

Comment:
Is this cache-able? Don't need to cache the static configs now, but can put a TODO for later perhaps.